exercises/pascals-triangle/test/test_pascals_triangle.c

```diff
@@ -0,0 +1,88 @@
+#include <stdbool.h>
+#include "../src/pascals_triangle.h"
+#include "vendor/unity.h"
+
+bool check(int count, int result[][count], int expected[][count])
+{
+   for (int i = 0; i < count; i++) {
+      for (int j = 0; j < i; j++) {
+         if (result[i][j] != expected[i][j]) {
+            return 0;
+         }
+      }
+   }
+   return 1;
+}
+
+void test_no_rows(void)
+{
+   int result[1][1];
+   int expected[1][1];
+   pascals_triangle(0, result);
+   TEST_ASSERT_TRUE(check(1, result, expected));
```

Comment:
I used my own little function because Unity does not have a 2d array test , but it has a 1d array test , i could use that with 'for loops' but i thought it would be light if i used this little check function